### DISCUSSION TITLE
Agenda meeting Wed Oct 2 2024

### DISCUSSION THREAD

**Discussion by megmcca** [coreen-mullen/fix_FitSNAP#2] [General]
**Super optional no stress pre-meeting task:**
Use the `git clone` command to clone your own copy of the [fix_fitsnap repo](https://github.com/coreen-mullen/fix_FitSNAP) somewhere (school cluster is probably best)
Note: for the time being, just having “git” and the fix_fitsnap clone itself is enough!
it does NOT need to function properly - you will NOT need LAMMPS, mpi4py, etc.
 
Meeting agenda:
- review of last meeting
- what is git vs. GitHub (vs. GitLab vs. BitBucket vs. SourceForge. …)
- Meg will walk through git branches, fetch/pull, status, commit, log, push
- GitHub issue(s)

**Comment by ifurrick** [comment]
Very demure post, Meg

**Comment by jgpasion** [comment]
demure

**Comment by hb20245** [comment]
ur mom
